main.go

```diff
@@ -126,6 +127,7 @@ func main() {
 		"Enable leader election for controller manager. "+
 			"Enabling this will ensure there is only one active controller manager.")
 	pflag.BoolVar(&createRBACPermissions, "create-rbac-permissions", false, "Automatically create RBAC permissions needed by the processors")
+	pflag.BoolVar(&enableMultiInstrumentation, "enable-multi-instrumentation", false, "Controls whether the operator supports multi instrumentation")
```

Comment:
@pavolloffay @jaronoff97 @TylerHelmuth are we ok with this naming convention? Ideally we should be consistent with the flag names, is `--enable-xxx-instrumentation` good?

Reply:
I think it is ok.  The individual language ones could be `enable-xxx-instrumentation`